src/appleseed.shaders/src/appleseed/as_triplanar.osl

```diff
@@ -81,7 +81,7 @@ shader as_triplanar
         string label = "Surface Point",
         string page = "Projection",
         int divider = 1,
-        int as_max_param_id = 3
+        int as_max_param_id = 2
```

Comment:
I don't think this is correct, @usakhelo has set the `as_max_param_id` values carefully as to preserve backward compatibility, and these shouldn't be changed.

Reply:
Hi,
I can confirm that we shouldn't change parameter IDs.
@luisbarrancos, I'm wondering if I shared these ID values with you already? They seem to be close to what I have in my branch but not exactly the same... I was wondering if it's something I shared with you at some point and then changed it.. Sorry, I don't remember anything.
Anyway, I have current IDs for all existing shaders in case we need it.

Reply:
Revert and use what's in master instead.

Reply:
@usakhelo i think you shared this with me when this PR was opened but it was some time ago and i don't remember it either. In any case, it should be ok now. Merged with the values in master.